tests/foreman/api/test_host.py

```diff
@@ -903,3 +910,151 @@ def test_negative_update_os(self):
             host = host.update(['operatingsystem'])
         self.assertNotEqual(
             host.read().operatingsystem.read().name, new_os.name)
+
+    @run_only_on('sat')
+    @stubbed
+    @tier3
+    def test_positive_create_baremetal_bios(self):
+        """Create a new Host from provided MAC address
+
+        @id: 9d74ed70-3197-4825-bf96-21eeb4a765f9
+
+        @steps:
+        1. Create a PXE-based VM with BIOS boot mode (outside of Satellite).
+        2. Create a new host using 'BareMetal' option and MAC address of the
+        pre-created VM
+
+        @assert: Host is created
+
+        @caseautomation: System
+
+        """
+
+    @run_only_on('sat')
+    @stubbed
+    @tier3
+    def test_positive_create_baremetal_uefi(self):
```

Comment:
how about:
`test_positive_create_baremetal_with_uefi` ?
i'd like to keep the `baremetal` keyword to stay compliant with already exisiting `test_positive_create_libvirt`. Also, i find the `mode` word useless.